backend/src/main/java/com/stampcrush/backend/application/manager/owner/ManagerCommandService.java

```diff
@@ -0,0 +1,44 @@
+package com.stampcrush.backend.application.manager.owner;
+
+import com.stampcrush.backend.application.manager.owner.dto.OwnerCreateDto;
+import com.stampcrush.backend.application.manager.owner.dto.OwnerLoginDto;
+import com.stampcrush.backend.auth.api.response.AuthTokensResponse;
+import com.stampcrush.backend.auth.application.util.AuthTokensGenerator;
+import com.stampcrush.backend.entity.user.Owner;
+import com.stampcrush.backend.exception.LoginIdBadRequestException;
+import com.stampcrush.backend.exception.OwnerNotFoundException;
+import com.stampcrush.backend.repository.user.OwnerRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class ManagerCommandService {
+
+    private final OwnerRepository ownerRepository;
+    private final AuthTokensGenerator authTokensGenerator;
+
+    public Long register(OwnerCreateDto ownerCreateDto) {
+        String loginId = ownerCreateDto.getLoginId();
+        String password = ownerCreateDto.getPassword();
+
+        if (isDuplicatedLoginId(loginId)) {
+            throw new LoginIdBadRequestException("중복되는 아이디 입니다.");
+        }
+        Owner owner = Owner.of(loginId, password);
+        return ownerRepository.save(owner).getId();
+    }
+
+    private boolean isDuplicatedLoginId(String loginId) {
+        return ownerRepository.findByLoginId(loginId).isPresent();
+    }
+
+    public AuthTokensResponse login(OwnerLoginDto ownerLoginDto) {
+        Owner owner = ownerRepository.findByLoginId(ownerLoginDto.getLoginId())
+                .orElseThrow(() -> new OwnerNotFoundException("존재하지 않는 아이디 입니다."));
```

Comment:
please write exception message specifically

Reply:
i agree. it does not make sense...

Reply:
leo : person in charge of logging error message